virttest/qemu_devices/qdevices.py

```diff
@@ -1667,62 +1763,75 @@ def _start_daemon(self):
         if self.get_param('extra_options'):
             fsd_cmd += self.get_param('extra_options')
 
-        logging.info('Running virtiofs daemon command %s.', fsd_cmd)
-        self._daemon_process = aexpect.run_bg(fsd_cmd, None,
-                                              self._handle_log,
-                                              auto_close=False)
-        status_active = 'Waiting for vhost-user socket connection'
-        self._daemon_process.read_until_any_line_matches(status_active,
-                                                         timeout=5)
-        logging.info("Created virtiofs daemon process with parent PID %d.",
-                     self._daemon_process.get_pid())
-
-    def _stop_daemon(self):
-        """Stop the virtiofs daemon."""
-        if self._daemon_process is not None:
-            try:
-                if not utils_misc.wait_for(lambda: not self._daemon_alive(), 3):
-                    raise DeviceError('The virtiofs daemon is still alive.')
-            finally:
-                self._daemon_process.close()
-                self._daemon_process = None
+        self.set_param('cmd', fsd_cmd)
+        self.set_param('run_bg_kwargs', {'output_func': self._handle_log,
+                                         'auto_close': False})
+        self.set_param('status_active', 'Waiting for vhost-user socket connection')
+        super(QVirtioFSDev, self).start_daemon()
 
-    def clear(self):
-        """Clear the virtiofs daemon."""
-        try:
-            self._stop_daemon()
-        except DeviceError:
-            pass
+    def __eq__(self, other, dynamic=True):
+        if super(QVirtioFSDev, self).__eq__(other, dynamic):
+            return self.get_param('sock_path') == other.get_param('sock_path')
+        return False
 
-    def hotplug(self, monitor):
-        self._start_daemon()
+    def __ne__(self, other):
+        return super(QVirtioFSDev, self).__ne__(other)
```

Comment:
The implementation of `__ne__` here should be unnecessary if it does nothing else, it should call its parent's function by default.

Reply:
Hi @vivianQizhu 
yes, we might not implement this part, but the CI checking will introduce a alter. I meet this issue too
when developing virtio-fs. So here must implement it or we ignore the alter.
please refer to https://github.com/avocado-framework/avocado-vt/pull/2460#issuecomment-614055643  for details.
Thanks.

Reply:
Yes, the documentation urges you to define __ne__ when you define __eq__, cause == does not always imply that != is false. but here you have already define __ne__ but just in the parent class, and the calling of the parent class function is unnecessary. So, here it makes sense that we ignore the alert.

Reply:
Hi @vivianQizhu 
All right, have updated. please help me to review, thanks.